server/queries/db.js

```diff
@@ -34,8 +34,17 @@ module.exports.createUser = username => {
     });
 }
 
-module.exports.getUser = (callback) => {
-    User.findAll().then(todos => callback(todos));
+module.exports.getUsers = () => {
+    return new Promise((resolve, reject) => {
+        User.findAll()
+        .then(users => resolve(users))
+        .catch(err => {
+            reject({
+                code: 404,
+                msg: err.message
+            });
+        });
+    });
 }
```

Comment:
If you havn't had a look at async await yet it might be a good time to have a play with it :) I'ts basically syntactic sugar for promises, but its nicer to read and looks synchronous. https://javascript.info/async-await

feel free to ignore this if you want to 👍  it would be something like:
```
module.exports.getUsers = async () => {
        try {
                const users = await User.findAll();
                return users;
        }catch(err) {
               throw({
                   code: 404,
                   msg: err.message
               });
        };

```